components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties

```diff
@@ -109,4 +111,7 @@ mandatory.claims.warning.msg.1=You need to provide consent for
 mandatory.claims.warning.msg.2=all the mandatory claims
 mandatory.claims.warning.msg.3=in order to proceed
 privacy.policy.cookies=Cookie Policy
+privacy.policy.cookies.short.description=After a successful sign in, we use a cookie in your browser to track your session. for more details you can refer our
```

Comment:
Shall we remove "for more details," section from the description here? Otherwise there is a duplicate phrase "for more details" coming from "privacy.policy.for.more.details" as well.

Reply:
Fixed with f52ed54